### PR TITLE
Filter in MemoriesFragment

### PR DESCRIPTION
**Pull request by Hicham010**
I added a navigation drawer and a filter feature by year and location.

### CODE REVIEW

**Review by MrTheGood — CHANGES_REQUESTED** [review]
Does not work. Trying to help, but this PR should NOT be accepted yet.

**Comment by MrTheGood** [comment]
I fixed the issue. **Do not forget to pull.**

The issue: 
in `MainActivity` we have the `NavHostFragment` view, in which all `Fragment`s are shown. This `NavHostFragment` did not have `layout_height` to `match_parent` and `DrawerLayout` apparently didn't like it. I cheated it a bit with some brute force (and lube) and now it works.


However, **This pull request should not be accepted yet as it's clearly not done.**

File: app/src/main/res/values-nl/strings.xml [hunk 1]
```diff
@@ -11,6 +11,10 @@
     <string name="str_enter_adress">Adres invullen</string>
     <string name="str_search">Zoeken…</string>
 
+    <!--Drawer-->
+    <string name="open">open</string>
+    <string name="close">close</string>
```
Comment (MrTheGood) [inline app/src/main/res/values-nl/strings.xml:16]: "close" is not a correct dutch translation.

File: app/src/main/res/menu/nav_menu.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        android:id="@+id/setting"
+        android:title="Setting">
+    </item>
+    <item
+        android:id="@+id/log"
+        android:title="LogOut">
+    </item>
+    </group>
+
+</menu>
```
Comment (MrTheGood) [inline app/src/main/res/menu/nav_menu.xml]: These items don't make sense.

**Review by MrTheGood — CHANGES_REQUESTED** [review]
This is clearly still unfinished.

**Review by MrTheGood — APPROVED** [review]
proxy duckduckgo-1

**Review by Almightywesley — APPROVED** [review]
Hicham this task was harder that we expected, but after our feedback and our help you finalu dit it. nice work! thumps up

**Pull request merged**